e107_plugins/pm/pm.php

```diff
@@ -882,7 +892,10 @@ function pm_user_lookup()
 	switch($action)
 	{
 		case 'send' :
-			$ns->tablerender(LAN_PM, $pm->breadcrumb(LAN_PLUGIN_PM_NEW) . $mes->render() . $pm->show_send($pm_proc_id));
+//--			$ns->tablerender(LAN_PM, $pm->breadcrumb(LAN_PLUGIN_PM_NEW) . $mes->render() . $pm->show_send($pm_proc_id));
+ 			$sc->bc_type = LAN_PM_55;
+   		$sc->bc_other = NULL;
+			$ns->tablerender(LAN_PM, $mes->render() . $pm->show_send($pm_proc_id));
 			break;
```

Comment:
Shouldn't $sc->bc_type = LAN_PLUGIN_PM_NEW here?

Reply:
@rica-carv I'd still like to get this merged. What are your thoughts on the comment above?

Reply:
@rica-carv any thoughts on Cameron's comment?

Reply:
@CaMer0n @Moc Sorry, i only see now your questions... @CaMer0n You're right, it should be PM_NEW...